### ISSUE TITLE
Active 'old-code' review

### ISSUE THREAD

**Issue by dalefrancis88** [Remitter/Discuss#4]
We are inheriting this code base and all the decisions that were made. Some of this were good, some bad and some were forced onto us. I know we've been working hard these last few months to get this up to scratch but what do we think about retroactively doing a code review of the code base and and changes or removing dead code?

**Comment by bijeebuss** [comment]
It would probably help new employees get up and running faster if we didn't have to weed through a bunch of code that isn't being used.

**Comment by dalefrancis88** [comment]
Yeah that's a fair point. Does it feel like theirs a lot of un-used or confusing code to you?

**Comment by bijeebuss** [comment]
No, but I'm sure I'll run into it eventually if there is a lot of it.

**Comment by afifmohammed** [comment]
I don't know of a more satisfying activity than deleting code. It can bite if we have loose ends hanging around once most of it is gone. As a proposal for next steps, we can open issues in respective repos qualifying all the dead code, and then systematically append PRs to them as we delete them one at a time. If deleting them all in the one go is not safe or is time consuming this can be a approach to ensure we get a start and continue to make progress.
